src/authenticated/project/ProjectEditPage.js

```diff
@@ -43,12 +45,16 @@ const ProjectEditPage = compose(
     props: ({ data: { Project } }) => ({ project: Project })
   }),
   graphql(updateProjectQuery, {
+    options: {
+      refetchQueries: ['getUser']
```

Comment:
Oh whoops, I am realizing `getUser` is the name of the query, not the type. I'm not sure if we want [`User` or `user`](https://github.com/codefordenver/members/blob/a9c1e136ccfca82257e7f452512eb95d46948635/src/authenticated/member/MemberProfilePage.js#L7). You may want to test both and see which one works

Reply:
Nevermind. It seems to be working like this.